internal/config/redirects.go

```diff
@@ -5,14 +5,18 @@ import (
 	"net/url"
 )
 
-type RedirectsStoreConfig struct {
+type GitlabRedirectsConfig struct {
+	Enabled        bool
 	RenkuBaseURL   *url.URL
 	RedirectedHost string
 }
+type RedirectsStoreConfig struct {
+	Gitlab GitlabRedirectsConfig
+}
 
 func (r *RedirectsStoreConfig) Validate() error {
-	if r.RenkuBaseURL == nil {
-		return fmt.Errorf("the redirects store config is missing the base url for Renku")
+	if r.Gitlab.Enabled && r.Gitlab.RenkuBaseURL == nil {
+		return fmt.Errorf("the redirects store is enabled but the config is missing the base url for Renku")
```

Comment:
A bit of a nitpick feel free to ignore if you want.

```suggestion
		return fmt.Errorf("the Gitlab redirects store is enabled but the config is missing the base url for Renku")
```